addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14100,7 +14100,29 @@ msgctxt "#36044"
 msgid "Disable polling for CEC capable devices?"
 msgstr ""
 
-#empty strings from id 36045 to 36100
+#. Desc of action to be taken when "changing CEC source" (1st)
+#. it is a tristate button in CEC adapter settings
+#: xbmc/peripherals/devices/PeripheralCecAdapter.cpp
+#: system/peripherals.xml
+msgctxt "#36045"
+msgid "No action"
+msgstr ""
+
+#. Desc of action to be taken when "changing CEC source" (2nd)
+#: xbmc/peripherals/devices/PeripheralCecAdapter.cpp
+#: system/peripherals.xml
+msgctxt "#36046"
+msgid "Keep audio, stop video"
```

Comment:
without further explanation it would not be obvious to me what this action actually does. It somehow reads as if video and GUI rendering would stop but f.e. the audio track of a video would still be played. Quite confusing wording IMO. Also, does it stop or pause video playback?

I think this settings needs to be handled different to your current appraoch. I'll outline my idea as direct comment and not as inline note.